init.sh

```diff
@@ -4,6 +4,13 @@ BRANCH=${REPO_BRANCH:='master'}
 INV=${INVENTORY:="/root/inventory.yaml"}
 VALS=${VALIDATIONS:=""}
 
+# Set logging env var to file mounted into the container.
+LOG_PATH="/root/validations.log"
```

Comment:
Do we really need to set a var for that? Since it seems to be used only in the block bellow... ?